# Awesome Protein Structure Prediction & Design [![Awesome](https://awesome.re/badge.svg)](https://awesome.re)

Working on it ⚒️

## Contents

- [Videos and Lectures](#videos-and-lectures)
- [Papers](#papers)
- [Datasets](#datasets)
- [Tools](#tools)
- [Miscellaneous](#miscellaneous)

## Videos and Lectures

- [2024 Nobel Prize lectures in chemistry | David Baker, Demis Hassabis and John Jumper](https://www.youtube.com/watch?v=HnT1VWzdFWc)
- [RosettaCommons Lectures](https://www.youtube.com/@RosettaCommons/videos)

## Papers
- [Chai-1: Decoding the molecular interactions of life](https://www.biorxiv.org/content/10.1101/2024.10.10.615955v1)
- [Simulating 500 million years of evolution with a language model](https://www.biorxiv.org/content/10.1101/2024.07.01.600583v1)
- [OpenFold: retraining AlphaFold2 yields new insights into its learning mechanisms and capacity for generalization](https://www.nature.com/articles/s41592-024-02272-z)
- [Multistate and functional protein design using RoseTTAFold sequence space diffusion](https://www.nature.com/articles/s41587-024-02395-w)
- [Protein interactions in human pathogens revealed through deep learning](https://www.nature.com/articles/s41564-024-01791-x)
- [De novo design of pH-responsive self-assembling helical protein filaments](https://www.nature.com/articles/s41565-024-01641-1)
- [Substrate interactions guide cyclase engineering and lasso peptide diversification](https://www.nature.com/articles/s41589-024-01727-w)
- [Opportunities and challenges in design and optimization of protein function](https://www.nature.com/articles/s41580-024-00718-y)
- [Self-driving laboratories to autonomously navigate the protein fitness landscape](https://www.nature.com/articles/s44286-023-00002-4)
- [Diffusion models in bioinformatics and computational biology](https://www.nature.com/articles/s44222-023-00114-9)
- [Score-based generative modeling for de novo protein design](https://www.nature.com/articles/s43588-023-00440-3)
- [Machine learning-enabled retrobiosynthesis of molecules](https://www.nature.com/articles/s41929-022-00909-w)
- [DiffDock: Diffusion Steps, Twists, and Turns for Molecular Docking](https://arxiv.org/abs/2210.01776)
- [Protein structure generation via folding diffusion](https://www.nature.com/articles/s41467-024-45051-2)
- [Language models of protein sequences at the scale of evolution enable accurate structure prediction](https://www.biorxiv.org/content/10.1101/2022.07.20.500902v1)
- [Computational design of soluble and functional membrane protein analogues](https://www.nature.com/articles/s41586-024-07601-y)
- [Accurate structure prediction of biomolecular interactions with AlphaFold 3](https://www.nature.com/articles/s41586-024-07487-w)
- [Generalized biomolecular modeling and design with RoseTTAFold All-Atom](https://www.science.org/doi/10.1126/science.adl2528)
- [Accurate prediction of protein–nucleic acid complexes using RoseTTAFoldNA](https://www.nature.com/articles/s41592-023-02086-5)
- [Predicting multiple conformations via sequence clustering and AlphaFold2](https://www.nature.com/articles/s41586-023-06832-9)
- [Clustering predicted structures at the scale of the known protein universe](https://www.nature.com/articles/s41586-023-06510-w) 
- [Computational design of transmembrane pores](https://www.nature.com/articles/s41586-020-2646-5)
- [De novo protein design by deep network hallucination](https://www.nature.com/articles/s41586-021-04184-w)
- [Role of backbone strain in de novo design of complex α/β protein structures](https://www.nature.com/articles/s41467-021-24050-7)
- [Design of protein-binding proteins from the target structure alone](https://www.nature.com/articles/s41586-022-04654-9)
- [Accurate computational design of three-dimensional protein crystals](https://www.nature.com/articles/s41563-023-01683-1)
- [Improving de novo protein binder design with deep learning](https://www.nature.com/articles/s41467-023-38328-5)
- [Uncovering new families and folds in the natural protein universe](https://www.nature.com/articles/s41586-023-06622-3) 
- [Protein structure prediction with in-cell photo-crosslinking mass spectrometry and deep learning](https://www.nature.com/articles/s41587-023-01704-z)
- [Towards a structurally resolved human protein interaction network](https://www.nature.com/articles/s41594-022-00910-8)
- [A structural biology community assessment of AlphaFold2 applications](https://www.nature.com/articles/s41594-022-00849-w)
- [Improved AlphaFold modeling with implicit experimental information](https://www.nature.com/articles/s41592-022-01645-6) 
- [ColabFold: making protein folding accessible to all](https://www.nature.com/articles/s41592-022-01488-1)
- [Accurate prediction of protein structures and interactions using a three-track neural network](https://www.science.org/doi/10.1126/science.abj8754)
- [Highly accurate protein structure prediction for the human proteome](https://www.nature.com/articles/s41586-021-03828-1) 
- [Highly accurate protein structure prediction with AlphaFold](https://www.nature.com/articles/s41586-021-03819-2)
- [Improved protein structure prediction using potentials from deep learning](https://www.nature.com/articles/s41586-019-1923-7) 
- [CombFold: predicting structures of large protein assemblies using a combinatorial assembly algorithm and AlphaFold2](https://www.nature.com/articles/s41592-024-02174-0) 
- [AlphaFold predictions are valuable hypotheses and accelerate but do not replace experimental structure determination](https://www.nature.com/articles/s41592-023-02087-4)
- [Improving AlphaFold2-based protein tertiary structure prediction with MULTICOM in CASP15](https://www.nature.com/articles/s42004-023-00991-6) 
- [Fast and effective protein model refinement using deep graph neural networks](https://www.nature.com/articles/s43588-021-00098-9)
- [De novo design of potent and selective mimics of IL-2 and IL-15](https://www.nature.com/articles/s41586-018-0830-7) 
- [Receptor subtype discrimination using extensive shape complementary designed interfaces](https://www.nature.com/articles/s41594-019-0224-z) 
- [De novo design of bioactive protein switches](https://www.nature.com/articles/s41586-019-1432-8)
- [Accurate _de novo_ design of hyperstable constrained peptides](https://www.nature.com/articles/nature19791) 
- [The coming of age of _de novo_ protein design](https://www.nature.com/articles/nature19946)
- [Computational design of self-assembling cyclic protein homo-oligomers](https://www.nature.com/articles/nchem.2673) 
- [_De novo_ design of a four-fold symmetric TIM-barrel protein with atomic-level accuracy](https://www.nature.com/articles/nchembio.1966)
- [Accurate design of co-assembling multi-component protein nanomaterials](https://www.nature.com/articles/nature13404) 
- [De novo design of a non-local β-sheet protein with high stability and accuracy](https://www.nature.com/articles/s41594-018-0141-6) 
- [De novo metalloprotein design](https://www.nature.com/articles/s41570-021-00339-5)
- [Constructing ion channels from water-soluble α-helical barrels](https://www.nature.com/articles/s41557-021-00688-0)
- [Language models enable zero-shot prediction of the effects of mutations on protein function](https://www.biorxiv.org/content/10.1101/2021.07.09.450648v1) 
- [MSA Transformer](https://www.biorxiv.org/content/10.1101/2021.02.12.430858v1) 
- [Biological Structure and Function Emerge from Scaling Unsupervised Learning to 250 Million Protein Sequences](https://www.biorxiv.org/content/10.1101/622803v4) 
- [Challenges in protein-folding simulations](https://www.nature.com/articles/nphys1713) 


## Datasets


## Tools


## Miscellaneous
- [The Baker Lab Podcast](https://www.bakerlab.org/podcast/)
- [Nature Portfolio Collection | Nobel Prize in Chemistry 2024](https://www.nature.com/collections/edjcfdihdi)
- [AIDrugX at NeurIPS24](https://sites.google.com/view/newmodality-aidrug/home)
- [Machine Learning in Structural Biology (Workshop at NeurIPS24)](https://www.mlsb.io/)
- [Boston Protein Design and Modeling Club](https://www.youtube.com/@bpdmc)
